### ISSUE TITLE
Remove fontwesome 

### ISSUE THREAD

**Issue by ExpressHermes** [ExpressHermes/JokesApp#3]
We are using a couple of icons from font awesome. The number is quite small and can be replaced with the SVGs. These svgs can be placed in the media directory and served directly from there in the app.

**Comment by saikalyan2002** [comment]
@ExpressHermes  ,Can I take this issue up?

**Comment by saikalyan2002** [comment]
@ExpressHermes  Hello!, Can I take up this issue?

**Comment by artimanputra** [comment]
Hey, I would like to work on this. Could you please assign it to me?

**Comment by SachinMittal1766** [comment]
Hi @artimanputra, are you still working on the issue?
If not, I would like to take this Issue @ExpressHermes

**Comment by artimanputra** [comment]
> Hi @artimanputra, are you still working on the issue? If not, I would like to take this Issue @ExpressHermes

Yup, I have created a PR for this as well. You can look for another issue.
